### ISSUE TITLE
Connected textures work only for the ctm_compact method, and only if the property used is matchTiles

### ISSUE THREAD

**Issue by Aeldit** [copycats-plus/copycats#51]
When using connected textures CTM-OF-Fabric (`0.1.2+1.20.x`) with Continuity (`3.0.0-beta.5+1.20.1`) and Copycats (`1.20.1-1.2.6`), the connected textures are not working as intended:

If the properties for the diamong block are as follows:

```properties
method=ctm_compact
tiles=0-4
matchTiles=diamond_block
```

the blocks connect properly between themselves (though they also connect to other copycats blocks even if they others don't have the same texture)
![2024-05-19_20 53 53](https://github.com/copycats-plus/copycats/assets/81476169/1cc15318-aaa0-484b-8a28-e918569bad3d)

But if we use `matchBlocks` instead of `matchTiles` or if we use the `ctm` method instead of the `ctm_compact` method, the textures no longer connect

(In red are 4 oak logs and in dark blue are 4 copycats blocks with oak log texture. The oak log connected textures use the `ctm` method, while the diamond block use the `ctm_compact` method)
![2024-05-19_20 58 16](https://github.com/copycats-plus/copycats/assets/81476169/745ad043-aaa7-40b3-9616-2d9c6df4578f)

So to summarize, the problem here is that all blocks using the `ctm` method or the `matchBlocks` property cannot connect
